### PR TITLE
Fixes for compiling //internal:tsc_wrapped with typescript 3.0+

### PR DESCRIPTION
**Pull request by Ubehebe**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature (please, look at the "Scope of the project" section in the README.md file)
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [x] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

//internal:tsc_wrapped does not compile with typescript 3.0+. 

## What is the new behavior?

//internal:tsc_wrapped compiles with typescript 3.0+.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: internal/tsetse/language_service_plugin.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@ function init() {
   return {
     create(info: ts.server.PluginCreateInfo) {
       const oldService = info.languageService;
-      const checker = new Checker(oldService.getProgram());
+      const checker = new Checker(oldService.getProgram()!);
```
Comment (mgechev) [inline internal/tsetse/language_service_plugin.ts]: If the program can be null in some cases, we may want to handle this case.

File: internal/tsetse/rules/ban_promise_as_condition_rule.ts [hunk 2]
```diff
@@ -94,7 +94,7 @@ function addFailureIfThenableCallExpression(
   }
 
   const typeChecker = checker.typeChecker;
-  const signature = typeChecker.getResolvedSignature(callExpression);
+  const signature = typeChecker.getResolvedSignature(callExpression)!;
```
Comment (mgechev) [inline internal/tsetse/rules/ban_promise_as_condition_rule.ts]: If the resolved signature could be null we may want to handle this.

**Comment by alexeagle** [comment]
I think it was already fixed in https://github.com/bazelbuild/rules_typescript/pull/348 ?

**Comment by alexeagle** [comment]
I believe this is already fixed

**Pull request closed without merging**